src/hotspot/os/windows/os_windows.cpp

```diff
@@ -4183,21 +4183,19 @@ jint os::init_2(void) {
 #endif
 
   // for debugging float code generation bugs
-  if (ForceFloatExceptions) {
-#ifndef  _WIN64
-    static long fp_control_word = 0;
-    __asm { fstcw fp_control_word }
-    // see Intel PPro Manual, Vol. 2, p 7-16
-    const long precision = 0x20;
-    const long underflow = 0x10;
-    const long overflow  = 0x08;
-    const long zero_div  = 0x04;
-    const long denorm    = 0x02;
-    const long invalid   = 0x01;
-    fp_control_word |= invalid;
-    __asm { fldcw fp_control_word }
+#if defined(_DEBUG) && !defined(_WIN64)
+  static long fp_control_word = 0;
+  __asm { fstcw fp_control_word }
+  // see Intel PPro Manual, Vol. 2, p 7-16
+  const long precision = 0x20;
+  const long underflow = 0x10;
+  const long overflow  = 0x08;
+  const long zero_div  = 0x04;
+  const long denorm    = 0x02;
+  const long invalid   = 0x01;
+  fp_control_word |= invalid;
```

Comment:
It looks like only `invalid` is used by the code. Are the other consts necessary?